support/ci/promote_rc.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+set -e
+
+if [ -z ${HAB_AUTH_TOKEN+x} ]; then
+  echo "HAB_AUTH_TOKEN var is unset"
+  exit 1
+fi
+if [ -z ${BINTRAY_USER+x} ]; then
+  echo "BINTRAY_USER var is unset"
+  exit 1
+fi
+if [ -z ${BINTRAY_KEY+x} ]; then
+  echo "BINTRAY_KEY var is unset"
+  exit 1
+fi
+
+HAB_VERSION=$(cat VERSION)
+PROMOTE_CHANNEL=stable
+RC_CHANNEL=rc-$HAB_VERSION
+RC_PKGS_JSON=$(curl https://bldr.habitat.sh/v1/depot/channels/core/$RC_CHANNEL/pkgs)
+RC_PKGS_JSON_DATA=$(echo $RC_PKGS_JSON | jq -c -r '.data | .[]')
+TOTAL_COUNT=$(echo $RC_PKGS_JSON | jq -r '.total_count')
+END=$(echo $RC_PKGS_JSON | jq -r '.range_end')
+
+if [ $(($END+1)) !=  $TOTAL_COUNT ]; then
+  echo "There are multipe pages of releases. Consider deleting the channel and then rebuild."
+  exit 1
+fi
+
+declare -A pkg_releases
+for pkg in $RC_PKGS_JSON_DATA
+do
+    ORIGIN=$(echo $pkg | jq -r ".origin")
+    NAME=$(echo $pkg | jq -r ".name")
+    VERSION=$(echo $pkg | jq -r ".version")
+    RELEASE=$(echo $pkg | jq -r ".release")
+    TARGET=$(echo $pkg | jq -r ".platforms[0]")
+    IDENT=$ORIGIN/$NAME/$VERSION/$RELEASE
+    if [[ -z ${pkg_releases[$NAME-$TARGET]+x} || "${pkg_releases[$NAME-$TARGET]}" < "$IDENT" ]]; then
```

Comment:
This logic merits a comment explaining why it's here. If we ever add options to the API to just return the latest releases from a channel, then it all goes away.

Reply:
☝️

Reply:
I put the comment just above the loop

Reply:
ah, so you did!